dpnp/backend/include/dpnp_iface_fptr.hpp

```diff
@@ -178,8 +178,6 @@ enum class DPNPFuncName : size_t
     DPNP_FN_FILL_DIAGONAL_EXT, /**< Used in numpy.fill_diagonal() impl, requires
                                   extra parameters */
     DPNP_FN_FLATTEN,           /**< Used in numpy.flatten() impl  */
-    DPNP_FN_FLATTEN_EXT,       /**< Used in numpy.flatten() impl, requires extra
```

Comment:
there is compilation fault, since `dpnp_krnl_elemwise.cpp` wasn't properly updated